airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -69,13 +68,13 @@
 
     @PostConstruct
     public void init() {
-        deleteOldIndexFiles();
+        indexManager.deleteOldIndexFiles();
```

Comment:
Re: https://github.com/airsonic/airsonic/pull/1166#issuecomment-515270408 and my own comment regarding keeping the old index when upgrading/downgrading airsonic versions, I agree with providing and correcting the `indexManager.deleteOldIndexFiles()` method, but I do not want it to be used automatically.

Reply:
This is probably inevitable.
As a system specification, regardless of the disk size, there is a debate as to whether to adopt rotate or not.
Some people hate rotation and some hate deletion.

I personally do not mind either, but I think that it may be taken into consideration that index schema changes are not frequent, and that some people prefer automation.

Reply:
Yes, it is probably inevitable, and I wouldn't have a problem with automatic cleanup once we are a few more release versions (of airsonic) ahead.  I just don't want it automatic when the change is so recent, since I often change between the last few versions to test things.

Reply:
I have made relevant comments on this matter.
It is necessary to strategically consider practical issues in technical issues and rather long-term maintenance, not emotional theory.

Unless there is anyone who can make revolutionary proposals, policy choices by the elimination method may be necessary.
What we can do is to present new information that will be used to make muffmanman decisions.

Reply:
Let's leave the automatic index deletion commented out for now. As per my previous comment, these index directories are not large in size. Therefore I see no penalty to keeping around the old index for a release or two and then adding automatic deletion in once the chance of a revert is much less.

Reply:
We have approved your request.